src/MsgProcessing.chpl

```diff
@@ -339,11 +340,38 @@ module MsgProcessing
                 repMsg = "set %s to %t".format(name, val);
             }
             when (DType.UInt64, DType.UInt64) {
+                // TODO create a .getBigIntValue()
```

Comment:
Is this referring to a different `.getBigIntValue` than the one created in `src/Message.chpl`?

Reply:
updated